tests/generate_template_reference_data.py

```diff
@@ -111,6 +111,16 @@ def init(project):
                 "bundle": [["mpi_op", "omp_op"]],
             },
         ],
+        "environments.incite.AndesEnvironment": [
+            {
+                "partition": ["batch", "gpu"],
+            },
+            {
+                "partition": ["batch"],
+                "parallel": [False, True],
+                "bundle": [["mpi_op", "omp_op"]],
+            },
+        ],
```

Comment:
When I try to run tests in `test_templates.py`, I get the following error:
```bash
E                   RuntimeError: No reference data for environment environments.incite.AndesEnvironment!

tests/test_templates.py:62: RuntimeError
```

The tests pass for the other environments. I thought adding this would tell the script to generate data for tests. Did I need to add anything else to make sure data is generated during these tests?

Reply:
@klywang Take a look at these instructions: https://docs.signac.io/projects/flow/en/latest/support.html#contributing-environments-to-the-package

If something is incomplete or unclear, please ask!

Reply:
It's pretty clear! Thank you.